climate_scanner/noise_classifier/body_classifier.py

```diff
@@ -0,0 +1,71 @@
+# -*- coding: utf-8 -*-
+
+import yaml
+# from climate_scanner.noise_classifier.data_utils import load_params
+from data_utils import load_params
+import os
+import spacy
+
+
+_ROOT = os.path.abspath(os.path.dirname(__file__))
+
+def get_full_path(path):
+    return os.path.join(_ROOT, path)
+
+
+# loading config params
+with open(os.path.join(_ROOT, 'data', "config.yml")) as f:
+    params = yaml.load(f, Loader=yaml.FullLoader)
+
+params = load_params()
+
+
+class BodyNonbodyClassifier:
+    """A class to define functions to train/test/predict and evaluate the noise classifier.
+    """
+
+    def __init__(self):
+        try:
+            self.model = spacy.load("en_noise_binary_model")
+            print(f"\nNoise model successfully loaded.")
+        except:
```

Comment:
Remember, if you are excepting a particular potetial error: catch that specific error rather than universally excepting.

e.g:

```
try:
    <do some desired action>

except FileNotfoundError:

      <have some specific handling for this expected error>

except KeyError:
     <different handling for a different expected error>

Reply:
This way you won't accidentally skip over errors that you need to see.